includes/amp-helper-functions.php

```diff
@@ -149,6 +149,14 @@ function () {
 		do_action( 'amp_plugin_update', $old_version );
 		AMP_Options_Manager::update_option( Option::VERSION, AMP__VERSION );
 	}
+
+	add_action(
```

Comment:
This action would be better suited in the `AMP_Options_Manager` class. You could create a static method `AMP_Options_Manager::init` and move the action there, then call that method here.